packages/writing-style/vale-styles/commercetools/WordList.yml

```diff
@@ -22,7 +22,8 @@ swap:
   Ajax: AJAX
   Android device: Android-powered device
   android: Android
-  application: app
+  app: application
+  apps: applications
```

Comment:
since you disallow "MC apps", "custom apps" in our product specific terminology, I suggest to allow "app" and "apps" as standalone terms - if we talk about iOS apps and Android apps in our documentation (eg. in the Swift SDK) that's perfectly fine.